cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/files/deploy/crds/csiscaleoperators.csi.ibm.com.cr.yaml

```diff
@@ -0,0 +1 @@
+../../../../../../../operator/ibm-spectrum-scale-csi-operator/deploy/crds/csiscaleoperators.csi.ibm.com.cr.yaml
```

Comment:
This file needs to be removed, it breaks cert

Reply:
Deleted